cms/grading/tasktypes/Communication.py

```diff
@@ -33,11 +33,10 @@
 from cms.grading.steps import compilation_step, evaluation_step_before_run, \
     evaluation_step_after_run, extract_outcome_and_text, \
     human_evaluation_message, merge_execution_stats, trusted_step
-from cms.grading.tasktypes import check_files_number
+from cms.grading.tasktypes import check_files_number, eval_output
 from . import TaskType, check_executables_number, check_manager_present, \
     create_sandbox, delete_sandbox, is_manager_for_compilation
 
-
```

Comment:
Super duper minor nit: please revert this; I think this is CMS's style to have 2 newlines after imports